policyengine_us/parameters/gov/states/ca/tax/income/alternative_minimum_tax/exemption/amount_child.yaml

```diff
@@ -0,0 +1,21 @@
+description: California provides this alternative minimum tax minimum exemption amount for children.  
+values:
+  2021-01-01: 7_850
+  2022-01-01: 8_300
+
+metadata:
+  label: California alternative minimum tax minimum child exemption amount 
+  period: year
+  unit: currency-USD
+  reference:
+    - title: 2021 Schedule P (540) Alternative Minimum Tax and Credit Limitations - Residents
+      href: https://www.ftb.ca.gov/forms/2021/2021-540-p.pdf#page=1
+    - title: 2022 Schedule P (540) Alternative Minimum Tax and Credit Limitations - Residents
+      href: https://www.ftb.ca.gov/forms/2022/2022-540-p.pdf#page=1
+    - title: 2022 Instructions for Schedule P (540) Line 22
+      href: https://www.ftb.ca.gov/forms/2022/2022-540-p-instructions.html
+    - title: 2021 Instructions for Schedule P (540) Line 22
+      href: https://www.ftb.ca.gov/forms/2021/2021-540-p-instructions.html
+    # The legal code does not specify the exemption amount
```

Comment:
i think this is based on [IRC 59(j)(1)(B)](https://www.law.cornell.edu/uscode/text/26/59#j_1_B)

more generally, only cite sources that corroborate the values